js/widgets/toolbar.js

```diff
@@ -47,7 +48,7 @@ return $.widget( "mobile.toolbar", {
 
 	_create: function() {
 		var leftbutton, rightbutton,
-			role = this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
+			role =  this.element.is( ":jqmData(role='header')" ) ? "header" : "footer",
```

Comment:
Still need to make this update and update usages of old way in this and fixed toolbar